src/invidious/search.cr

```diff
@@ -463,5 +231,20 @@ def process_search_query(query, page, user, region)
     count, items = search(search_query, search_params, region).as(Tuple)
   end
 
-  {search_query, count, items, operators}
+  # Light processing to flatten search results out of Categories.
+  # They should ideally be supported in the future.
```

Comment:
What do you mean by that? I've seen that you made a similar chunk of code in `extract_videos()`, maybe you could make a function for that, or include it directly in your extractor code?

Reply:
![categories](https://user-images.githubusercontent.com/70992037/127895839-4e287582-ac5a-45b6-bd6e-7db5cda093f7.png)

What you see here are YouTube categories. The current Invidious UI doesn't support them so we'd need to extract out the contents of the categories instead.

Reply:
Oh, okay. For the second category ("people also watched"), I think we should discuss if we add that or not, as it is kind of an anti-feature (it's attention gathering content, not very relevant to the searched terms).